app/javascript/articles/components/ArticleCoverImage.jsx

```diff
@@ -3,14 +3,22 @@ import { articlePropTypes } from '../../common-prop-types';
 
 export const ArticleCoverImage = ({ article }) => {
   return (
-    <a
-      href={article.path}
-      className="crayons-story__cover"
-      title={article.title}
-      style={{ backgroundImage: `url(${article.main_image})` }}
-    >
-      <span class="hidden">{article.title}</span>
-    </a>
+    <div className="crayons-article__cover crayons-article__cover__image__feed">
+      <a
+        href={article.path}
+        className="crayons-article__cover__image__feed crayons-story__cover__image"
+        title={article.title}
+      >
+        <img
+          className="crayons-article__cover__image__feed"
+          src={article.main_image}
+          width="650"
+          height="275"
+          alt={`Cover for ${article.title}`}
```

Comment:
```suggestion
          alt={article.title}
```
Since the image makes up the accessible name for the link, it makes more sense to just have the article title here (since that's where the link goes)

I'm curious why we have a `hidden` article title in a span too, but it's not in scope of this PR and can remain a mystery for now 😄